lib/testutils/httpmultibin/httpmultibin.go

```diff
@@ -395,6 +481,9 @@ func NewHTTPMultiBin(t testing.TB) *HTTPMultiBin {
 			"HTTP2BIN_PORT", http2URL.Port(),
 
 			"GRPCBIN_ADDR", fmt.Sprintf("%s:%s", httpsDomain, http2URL.Port()),
+			"LOCALHOST_CERT", strings.ReplaceAll(localhostCert, "\n", "\\n"),
+			"LOCALHOST_KEY", strings.ReplaceAll(localhostKey, "\n", "\\n"),
+			"LOCALHOST_ENCRYPTED_KEY", strings.ReplaceAll(localhostEncryptedKey, "\n", "\\n"),
```

Comment:
Isn't this only used for the gRPC test - it isn't even used anywhere in the actuall httpmultibin for anything?

Reply:
Correct.

That said, GRPCBIN_ADDR is also only used in runner_test.go and grpc/client_test.go. I thought those Replacer entries would fit there. I'm open to other suggestions.

I can modify the httpmultibin to support "appending" Replacer entries. I'm not certain what the additional impact would be.

Reply:
I've reverted the changes I made to httpmultibin.go and inlined my test data for my changes to a separate TestClient function within client_test.go